mws/resource_mws_customer_managed_keys.go

```diff
@@ -148,7 +148,8 @@ func ResourceCustomerManagedKeyV0() cty.Type {
 							Type: schema.TypeString,
 						},
 						"key_alias": {
-							Type: schema.TypeString,
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
it's not required - it's a schema for previous version of the state.

Reply:
I see, will revert. Thanks!